<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveTemplateConfigPage</class>
 <widget class="QWidget" name="CurveTemplateConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QListView" name="templateList"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupTemplate">
     <property name="title">
      <string>Template</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>editName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="editName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblLineStyle">
        <property name="text">
         <string>Line Style</string>
        </property>
        <property name="buddy">
         <cstring>comboLineStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineStyleCombo" name="comboLineStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinLineWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSymbolStyle">
        <property name="text">
         <string>Symbol Style</string>
        </property>
        <property name="buddy">
         <cstring>comboSymbolStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SymbolStyleCombo" name="comboSymbolStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinSymbolSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblErrorBarStyle">
        <property name="text">
         <string>Error Bar Style</string>
        </property>
        <property name="buddy">
         <cstring>comboErrorBarStyle</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineStyleCombo" name="comboErrorBarStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinErrorBarWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupTemplateByType">
     <property name="title">
      <string>Template for Data Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblExperimentalCurve">
        <property name="text">
         <string>Experimental Curve</string>
        </property>
        <property name="buddy">
         <cstring>comboExperimentalCurveTemplate</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboExperimentalCurveTemplate"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboProbabilityCurveTemplate"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblProbabilityCurve">
        <property name="text">
         <string>Probablity Curve</string>
        </property>
        <property name="buddy">
         <cstring>comboProbabilityCurveTemplate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboTheoreticalCurveTemplate"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTheoreticalCurve">
        <property name="text">
         <string>Theoretical Curve</string>
        </property>
        <property name="buddy">
         <cstring>comboTheoreticalCurveTemplate</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineStyleCombo</class>
   <extends>QComboBox</extends>
   <header>saxsview_linestylecombo.h</header>
  </customwidget>
  <customwidget>
   <class>SymbolStyleCombo</class>
   <extends>QComboBox</extends>
   <header>saxsview_symbolstylecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>comboLineStyle</tabstop>
  <tabstop>spinLineWidth</tabstop>
  <tabstop>comboSymbolStyle</tabstop>
  <tabstop>spinSymbolSize</tabstop>
  <tabstop>comboErrorBarStyle</tabstop>
  <tabstop>spinErrorBarWidth</tabstop>
  <tabstop>comboExperimentalCurveTemplate</tabstop>
  <tabstop>comboTheoreticalCurveTemplate</tabstop>
  <tabstop>comboProbabilityCurveTemplate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
